Intermediate-concurrency-tutorial/src/main/java/com/vinnik/kate/TaskExecutor.java

```diff
@@ -0,0 +1,114 @@
+package com.vinnik.kate;
+
+import java.util.Random;
+
+/** This class starts threads, which are measures time of processing tasks. */
+public final class TaskExecutor {
+    private static final int UPPER_BOUND_OF_RANDOM = 10000;
+    private static final int NANOSECONDS_IN_MILLISECONDS = 1000000;
+
+    private final int amountOfThreads;
+
+    private final TaskTable taskTable;
+    private final ResultTable resultTable;
+
+    private boolean resultTableIsBusy;
```

Comment:
In general, variables like this are named "isResultTableBusy" (like question) and/or "resultTableOccupationFlag".